x/staking/keeper/grpc_query.go

```diff
@@ -0,0 +1,417 @@
+package keeper
+
+import (
+	"context"
+	"strings"
+
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
+
+	"github.com/cosmos/cosmos-sdk/store/prefix"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/types/query"
+	"github.com/cosmos/cosmos-sdk/x/staking/types"
+)
+
+type Querier struct {
```

Comment:
can you add a comment on why the Keeper needs to be wrapped?

Reply:
Sure